Cargo.toml

```diff
@@ -56,6 +56,7 @@ c8y_http_proxy = { path = "crates/extensions/c8y_http_proxy" }
 c8y_log_manager = { path = "crates/extensions/c8y_log_manager" }
 c8y_mapper_ext = { path = "crates/extensions/c8y_mapper_ext" }
 camino = "1.1"
+cap = "0.1"
```

Comment:
Side commentary: Came across this [sysinfo](https://docs.rs/sysinfo/latest/sysinfo/) crate that covers a lot more parameters like CPU usage. Something we can consider in the future, for advanced monitoring.

Reply:
The purpose is different. Sysinfo observes the operating system, while here the point is to monitor the amount of  memory actually allocated by tedge on the heap.